server/db/index.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const petCardSchema = new mongoose.Schema({
+    Name: String,
+    Age: Number,
+    Type: String,
+    Color: String,
+
+});
```

Comment:
this file should be under api/pet/pet.model.js

you always want to ware in feature-oriented. having all the files that connected under the same directory. 
so you should have under api/pet/
1. routes.js
2. controller.js
3.model.js
4.test.js 

etc

Reply:
Fixed